frameworks/kafka/tests/test_sanity.py

```diff
@@ -47,20 +47,10 @@ def test_service_health():
 @pytest.mark.sanity
 def test_endpoints_address():
     foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
+    endpoints = sdk_networks.wait_for_endpoint_info(
```

Comment:
Could we do more to align this with the TensorFlow test that you added?

Reply:
I think most of the data-services do a count check for the endpoints. Thus it might be applicable for more than the Kafka service. In this case, it already looks quite similar because I started the Tensorflow implementation based on this. I'd rather have a look at this in a follow-up PR where I try to reuse more of the existing Tensorflow endpoint test util.